src/pages/webhooks/index.md

```diff
@@ -7,370 +7,30 @@ keywords:
-| `required`             | Bool   | Sets whenever the hook should be required or optional. Optional hook execution failure is logged and the process or the batches of hooks continues. Required hook execution failure terminates the process. | false       | true    |
-| `timeout`              | Int    | A hard timeout limit (milliseconds) for the request. Requests exceeding this timeout will be aborted with errors in the logs                                                                                | false       | 0       |
-| `softTimeout`          | Int    | A soft timeout limit (milliseconds) for the request. Requests exceeding this timeout will be logged for debugging purposes.                                                                                 | false       | n/a     |
-| `fallbackErrorMessage` | Int    | The error message that will be used in case of failed hooks execution.                                                                                                                                      | false       | n/a     |
-| `remove`               | Bool   | Removed hook is skipped during the batch execution.                                                                                                                                                         | false       | false   |
-
-#### Hook fields attributes:
-
-| Attribute   | Type    | Description                                                                           | Is required | Default |
-|-------------|---------|---------------------------------------------------------------------------------------|-------------|---------|
-| `name`      | String  | The field name or path of the value.                                                  | true        | n/a     |
-| `source`    | String  | The path to the value in the payload. If not set the `name` is used as source.        | false       | n/a     |
-| `converter` | String  | A class that is used to convert the value of the field before sending to the webhook. | false       | n/a     |
-| `remove`    | Boolean | Set to true to remove the field from the payload.                                     | false       | false   |
-
-#### Hook headers attributes:
-
-| Attribute  | Type    | Description                                                  | Is required | Default |
-|------------|---------|--------------------------------------------------------------|-------------|---------|
-| `name`     | String  | The header name will be in the same form as it will be sent. | false       | n/a     |
-| `resolver` | String  | The resolver class that appends headers dynamically.         | false       | n/a     |
-| `remove`   | Boolean | Set to true to remove the header from the request.           | false       | false   |
+1. [Test](testing.md) your `webhooks.xml` file and understand the [response](responses.md).
```

Comment:
Is "understand" the right word here? maybe they should "validate" the response or something similar.